smartapps/smartthings/keep-me-cozy-ii.src/i18n/ar-AE.properties

```diff
@@ -20,3 +20,5 @@
 '''Next page'''=الصفحة التالية
 '''Text'''=النص
 '''Number'''=الرقم
+'''CHOOSE THERMOSTAT…'''=اختيار الثرموستات...
```

Comment:
the `...` is the special unicode character rather than the 3 single periods

Reply:
also the text is all caps instead of how it is in the code, not sure if that will work